build.gradle

```diff
@@ -50,243 +50,7 @@ ext.getPlatformPath = { binary ->
-        if (project.includeJava) {
-            compileTask.dependsOn "unzipOpenCvJni_${openCvPlatform}"
-        }
-        if (targetPlatform.architecture.arm) {
-            linker.args "${openCvNativesFolder}/libopencv_calib3d.so"
-            linker.args "${openCvNativesFolder}/libopencv_features2d.so"
-            linker.args "${openCvNativesFolder}/libopencv_flann.so"
-            linker.args "${openCvNativesFolder}/libopencv_highgui.so"
-            linker.args "${openCvNativesFolder}/libopencv_imgcodecs.so"
-            linker.args "${openCvNativesFolder}/libopencv_imgproc.so"
-            linker.args "${openCvNativesFolder}/libopencv_ml.so"
-            linker.args "${openCvNativesFolder}/libopencv_objdetect.so"
-            linker.args "${openCvNativesFolder}/libopencv_photo.so"
-            linker.args "${openCvNativesFolder}/libopencv_shape.so"
-            linker.args "${openCvNativesFolder}/libopencv_stitching.so"
-            linker.args "${openCvNativesFolder}/libopencv_superres.so"
-            linker.args "${openCvNativesFolder}/libopencv_video.so"
-            linker.args "${openCvNativesFolder}/libopencv_videoio.so"
-            linker.args "${openCvNativesFolder}/libopencv_videostab.so"
-            linker.args "${openCvNativesFolder}/libopencv_core.so"
-            linker.args "-ldl"
-            linker.args "-lz"
-        } else if (targetPlatform.operatingSystem.windows) {
-            linker.args "${openCvNativesFolder}/opencv_core.lib"
-        } else {
-            linker.args "${openCvNativesFolder}/libopencv_calib3d.a"
-            linker.args "${openCvNativesFolder}/libopencv_features2d.a"
-            linker.args "${openCvNativesFolder}/libopencv_flann.a"
-            linker.args "${openCvNativesFolder}/libopencv_highgui.a"
-            linker.args "${openCvNativesFolder}/libopencv_imgcodecs.a"
-            linker.args "${openCvNativesFolder}/libopencv_imgproc.a"
-            linker.args "${openCvNativesFolder}/libopencv_ml.a"
-            linker.args "${openCvNativesFolder}/libopencv_objdetect.a"
-            linker.args "${openCvNativesFolder}/libopencv_photo.a"
-            linker.args "${openCvNativesFolder}/libopencv_shape.a"
-            linker.args "${openCvNativesFolder}/libopencv_stitching.a"
-            linker.args "${openCvNativesFolder}/libopencv_superres.a"
-            linker.args "${openCvNativesFolder}/libopencv_video.a"
-            linker.args "${openCvNativesFolder}/libopencv_videoio.a"
-            linker.args "${openCvNativesFolder}/libopencv_videostab.a"
-            linker.args "${openCvNativesFolder}/libopencv_core.a"
-            linker.args "${openCvNativesFolder}/liblibjpeg.a"
-            linker.args "${openCvNativesFolder}/liblibpng.a"
-            linker.args "-ldl"
-            linker.args "-lz"
-        }
-    }
-}
+apply from: "dependencies.gradle"
 
 ext.setupDefines = { project, binaries ->
     binaries.all {
```

Comment:
The addWpiUtilLibraryLinks on line 63 needs to be updated to use shared libs on arm (e.g. `if (targetPlatform.architecture.arm)` and static libs on other platforms.